packages/webview_flutter/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -302,7 +302,9 @@ Future<void> main() async {
       isPaused =
           await controller.runJavaScriptReturningResult('isPaused();') as bool;
       expect(isPaused, true);
-    });
+    },
+        // TODO(bparrishMines): Stop skipping once https://github.com/flutter/flutter/issues/148487 is resolved
```

Comment:
@bparrishMines I assigned the TODO to you as code owner since it wouldn't allow me to leave it empty.